must-gather/collection-scripts/gather

```diff
@@ -17,21 +20,36 @@ group_resources=()
 # Get namespace of node-maintenance-operator - where it is installed
 NMO_NAMESPACE=$(oc get subs -A --field-selector=metadata.name=node-maintenance-operator -o jsonpath='{.items[*].metadata.namespace}')
 
-# Get nmo logs - Nodes, CRD, and CRs
+# Get NMO Information - nodes' object, CRD, pod's logs, CRs, cluster's resource usage, and nodes' logs
+
+# Get Nodes' names
+NODES_NAMES=($(oc get nodes -o jsonpath='{.items[*].metadata.name}{"\n"}'))
 
-# Nodes
+# Nodes' objects
 group_resources+=(nodes)
 
-# NMO CRD
+# NMO's CRD
 NMO_CRD=$(oc get crds -o jsonpath='{range .items[*]}{"crd/"}{.metadata.name}{"\n"}{end}' | grep 'nodemaintenance.medik8s' | sed -z 's/\n/ /g')
 named_resources+=(${NMO_CRD})
 
-# node maintenance CRs
+# NMO's POD name
+NMO_POD=$(oc get pods -o jsonpath='{range .items[*]}{"pod/"}{.metadata.name}{"\n"}{end}' -n ${NMO_NAMESPACE}  | grep 'node-maintenance')
+
+# Node Maintenance CRs
 group_resources+=(nm)
 
 # Run the Collection of Resources using inspect
 oc adm inspect --dest-dir must-gather --all-namespaces "${named_resources[@]}"
 group_resources_text=$(IFS=, ; echo "${group_resources[*]}")
 oc adm inspect --dest-dir must-gather --all-namespaces "${group_resources_text}"
 
+# Get pod's logs 
+echo "$(oc logs ${NMO_POD} -n ${NMO_NAMESPACE})" > must-gather/operator-pod-logs/${OPERATOR_NAME}
+
+# Get cluster's resource (CPU and Memory) usage 
+echo "$(oc adm top node --use-protocol-buffers)" > must-gather/cluster_resoruce_statistics
+
+# Get all journal logs from each Node (TODO: select particluar journal logs, e.g., crio )
+for NODE in ${NODES_NAMES[@]}; do oc adm node-logs ${NODE} > must-gather/nodes-logs/${NODE}; done
```

Comment:
What is the usecase for node logs?
Not saying that it's bad, but be aware that it potentially can increase time for running must-gather and must-gather file size dramatically in big clusters with hundreds of nodes.

Reply:
Now It takes few seconds for six nodes... I couldn't find something important which is missing, but it is also hard to look for something when the node's logs can be noisy.
I will comment it for now, and maybe in the future I will add something more concrete.